services/web/server/src/simcore_service_webserver/rest_healthcheck.py

```diff
@@ -0,0 +1,93 @@
+""" Service healthcheck
+
+From https://docs.docker.com/engine/reference/builder/#healthcheck
```

Comment:
Honestly I thought this would be describing the below implementation, but it does not look like it.

Reply:
Now it does. Please read again.